global-transaction-service.env

```diff
@@ -0,0 +1,28 @@
+MONGO_PORT=27017
```

Comment:
Is this file used anywhere?

Reply:
Here: https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/4dad79fa35a56c6932b13b51ca68d4ae22823eb2/.devops/code-review-pipelines.yml#L81